app/api/analytics.py

```
@@ -0,0 +1,413 @@
+):
+    """Get aggregated sentiment metrics for entities"""
+    try:
+        metrics = await analytics_service.get_aggregated_sentiment_metrics(
+            entity_type=entity_type,
+            entity_id=entity_id,
+            start_date=start_date,
+            end_date=end_date,
+            sources=sources,
+            categories=categories,
+        )
+        return metrics
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)